app/views/layouts/application.html.haml

```diff
@@ -47,6 +47,10 @@
   = succeed '.' do
     = link_to "Find out more about cookies", cookies_path
 
+- content_for :after_header do
```

Comment:
Happy to make that change if it's only used for that view and not shared anywhere else. I was copying the original changes from #138.